index.html

```diff
@@ -38,5 +37,131 @@ <h1>
+                <li class="language">javascript</li>
+            </ul>
+
+            <button class="btn">See Project</button>
+        </article>
+
+        <article class="project">
+            <img src="./img/project2.png" alt="">
+            <h2 class="project-title">Multi-Post Stories</h2>
+            <div class="stats">
+                <span class="company">CANOPY</span>
+                <span class="role-and-year">Back End Dev</span>
+                <span class="role-and-year">2015</span>
+            </div>
+            <p>
+                A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </p>
+
+            <ul class="languages">
+                <li class="language">html</li>
+                <li class="language">css</li>
+                <li class="language">javascript</li>
+            </ul>
+
+            <button class="btn">See Project</button>
+        </article>
+
+        <article class="project">
+            <img src="./img/project3.png" alt="">
+            <h2 class="project-title">Tonic</h2>
+            <div class="stats">
+                <span class="company">CANOPY</span>
+                <span class="role-and-year">Back End Dev</span>
+                <span class="role-and-year">2015</span>
+            </div>
+            <p>
+                A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </p>
+
+            <ul class="languages">
+                <li class="language">html</li>
+                <li class="language">css</li>
+                <li class="language">javascript</li>
+            </ul>
+
+            <button class="btn">See Project</button>
+        </article>
+
+        <article class="project">
+            <img src="./img/project4.png" alt="">
+            <h2 class="project-title">Multi-Post Stories</h2>
+            <div class="stats">
+                <span class="company">CANOPY</span>
+                <span class="role-and-year">Back End Dev</span>
+                <span class="role-and-year">2015</span>
+            </div>
+            <p class="description">
+                A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </p>
+
+            <ul class="languages">
+                <li class="language">html</li>
+                <li class="language">css</li>
+                <li class="language">javascript</li>
+            </ul>
+
+            <button class="btn">See Project</button>
+        </article>
+    </section>
+
+    <section class="about-me">
+        <h1>
+           About Myself
+        </h1>
+
+        <p>
+            Hello I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+        </p>
+        <a href="#" class="connect-me">LET'S CONNECT</a>
```

Comment:
- [ ] Since this `Let's connect` is referring to the following social media icons, then it should be header using `hx` tag

Reply:
Well resolved the issue!